app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,23 +1,35 @@
 module Api
   module V1
-    class SessionsController < ApplicationController
-      skip_before_action :authenticate_user!, only: %i[create]
-
+    class SessionsController < ApiController
       def create
-        result = LoginUser.call(session_params)
+        result = LoginUser.call(session_creating_params)
 
         if result.success?
-          render json: { token: result.token }
+          render json: result.to_h.slice(:access_token, :refresh_token)
```

Comment:
You're planning to return refresh token in the response if I understand correctly. Is that right?

It means frontend developers should store refresh token in their side, which is not secure. Because third-party services can steal token using JS and send a request with an access token and refresh token to our API app and receive a new pair of tokens. What do you think about storing refresh token in the HTTP-only cookie? In this case, FE developers will not have access to this cookie. Also, third-party services will not be able to steal this cookie from the FE application using JS. 
Those, you will set a cookie with a refresh token, and send this cookie in response. FE developers will not read this cookie, but they will always send all existing cookies to our server then they will do request. So, we can read cookie with the refresh token and update the access and refresh tokens. What do you think about it?